### PR TITLE
[Deno] Generate Project

### PR DESCRIPTION
**Pull request by vyktoremario**
## Type of change

- [ ] Documentation change
- [ ] Bug fix
- [x] New Kit setup

## Checklist

- [x] These changes follow the [contributing guidelines](https://github.com/thisdot/starter.dev/blob/main/CONTRIBUTING.md)
- [x] This starter kit has been approved by the maintainers
- [x] Changes for this new starter kit are being pushed to an integration branch instead of main
- [x] All dependencies are version locked
- [ ] This fix resolves #
- [ ] I have verified the fix works and introduces no further errors

### CODE REVIEW

**Review by dariodjuric — CHANGES_REQUESTED** [review]
I would also suggest to add a lock file that locks the Oak version.

https://deno.land/manual@v1.28.2/basics/modules/integrity_checking

File: starters/deno-oak-denodb/src/main_test.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { assertEquals, testing } from '../dev_deps.ts';
+
+Deno.test(function addTest() {
```
Comment (dariodjuric) [inline starters/deno-oak-denodb/src/main_test.ts:3]: Do we need this test or can we hold off with dummy tests until we have something real to test? 🤔

Comment (dariodjuric) [reply starters/deno-oak-denodb/src/main_test.ts:3]: I was just suggesting to remove it as it doesn't provide any value, but if you think we should keep it, let's keep it.

Comment (vyktoremario) [reply starters/deno-oak-denodb/src/main_test.ts:3]: I can remove it then.

Comment (vyktoremario) [reply starters/deno-oak-denodb/src/main_test.ts]: Done

**Review by dariodjuric — APPROVED** [review]

**Pull request merged**